src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1461,6 +1503,53 @@ private RefreshItem getRefreshToken(final AuthenticationRequest request) {
         return refreshItem;
     }
 
+    private TokenCacheItem findFamilyItemForUser(final UserIdentifierType userIdentifierType, final String userId) {
+        final String methodName = ":findFamilyItemForUser";
+            
+        if (StringExtensions.IsNullOrBlank(userId)) {
+            Logger.v(TAG + methodName, "User id is not provided, cannot continue with finding family item.");
+            return null;
+        }
+
+        Iterator<TokenCacheItem> allItems = mTokenCacheStore.getAll();
```

Comment:
I think it would be much better if TokenCacheStore supports `Iterable<T>` - so you don't need to rebuild all iteration logic yourself.